README.md

```
@@ -1,4 +1,4 @@
-# Synix Bot
+ # Synix Bot
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)